### ISSUE TITLE
[.5][ariesjia] Conform Height of Data-Empty and Data-Full Trades Window

### ISSUE THREAD

**Issue by landry314** [bitshares/bitshares-ui#385]
When there is no data being displayed in the Trades window in a particular pair then the window is shorter than when it is full of data. When it is shorter (empty) it matches the height of Buy and Sell but when it is longer (full) it creates a gap.

Steps to Reproduce:

1) Find a pair in which you have made no trades
2) Click on My Trades which should make it empty
3) Click on Market Trades which should be full

See the height change?

Basically, the height should always conform with the Buy and Sell Windows because that looks right. The actual bug here is when it is full it is taking up a little more room and pushing the next row down.

**Comment by wmbutler** [comment]
Easy fix. Added to October.

![385](https://user-images.githubusercontent.com/1254810/30327561-1f3926dc-9792-11e7-8e61-7dca59f73f0e.gif)

**Comment by ariesjia** [comment]
can i take this bug

**Comment by wmbutler** [comment]
Yep.

**Comment by ariesjia** [comment]
Yep, It' s easy to fix it .

**Comment by calvinfroedge** [comment]
Closing this issue as it was merged
